test/appium/tests/atomic/chats/test_commands.py

```diff
@@ -228,10 +228,94 @@ def test_contact_profile_send_transaction(self):
         chat_view.view_profile_button.click()
         chat_view.profile_send_transaction.click()
         chat_view.chat_message_input.click()
-        chat_view.eth_asset.click()
+        chat_view.asset_by_name('ETH').click()
         amount = chat_view.get_unique_amount()
         chat_view.send_as_keyevent(amount)
         chat_view.send_message_button.click()
         send_transaction_view = chat_view.get_send_transaction_view()
         send_transaction_view.sign_transaction(common_password)
         self.network_api.find_transaction_by_unique_amount(recipient['address'], amount)
+
+    @marks.testrail_id(3744)
+    def test_send_tokens_in_1_1_chat(self):
+        recipient = transaction_users['D_USER']
+        sender = transaction_users['C_USER']
+        self.create_drivers(2)
+        device_1, device_2 = SignInView(self.drivers[0]), SignInView(self.drivers[1])
+        home_1 = device_1.recover_access(passphrase=sender['passphrase'], password=sender['password'])
+        home_2 = device_2.recover_access(passphrase=recipient['passphrase'], password=recipient['password'])
+        wallet_1, wallet_2 = home_1.wallet_button.click(), home_2.wallet_button.click()
+        wallet_1.set_up_wallet()
+        wallet_1.home_button.click()
+        wallet_2.set_up_wallet()
+        wallet_2.home_button.click()
+
+        chat_1 = home_1.add_contact(recipient['public_key'])
+        amount = chat_1.get_unique_amount()
+        chat_1.send_transaction_in_1_1_chat('STT', amount, sender['password'])
+
+        if not chat_1.chat_element_by_text(amount + '.STT').is_element_displayed():
+            self.errors.append('Message with the sent amount is not shown for the sender')
+        chat_2 = home_2.get_chat_with_user(sender['username']).click()
+        if not chat_2.chat_element_by_text(amount + '.STT').is_element_displayed():
+            self.errors.append('Message with the sent amount is not shown for the recipient')
```

Comment:
the verify seems to be broken since in https://github.com/status-im/status-react/pull/5041#issuecomment-402820528 it failed, but the amount is shown. Might be an issue with locator